packages/cli/src/databases/repositories/folder.repository.ts

```diff
@@ -213,7 +213,9 @@ export class FolderRepository extends Repository<FolderWithWorkflowCount> {
 		direction: 'DESC' | 'ASC',
 	): void {
 		if (field === 'name') {
-			query.orderBy('LOWER(folder.name)', direction);
+			query
+				.addSelect('LOWER(folder.name)', 'folder_name_lower')
```

Comment:
Fixes bug that found that throws error if sortBy name without explicitly selecting name